### ISSUE TITLE
Set current BE user as author for new posts

### ISSUE THREAD

**Issue by fnagel** [fnagel/t3extblog#114]
Seems only possible by using some TYPO3 API hooks or by adding TCAdefaults for each user.

Related links:
https://forge.typo3.org/issues/17745
https://forge.typo3.org/issues/36338

**Comment by mgrn0** [comment]
just curious since the documentation says

> Author: is determined by the current BE user

is this supposed to be working or not (yet)?

**Comment by fnagel** [comment]
No, it's not working. I guess that's just an error in the docs. Sorry for the confusion. Pushed a fix to 5.x and master branch. Thanks for asking / reporting!

**Comment by mgrn0** [comment]
Thanks for the clarification!
